regex-assembly/include/unix-shell-4andup-with-params.ra

```diff
@@ -4,13 +4,32 @@
 ##! This list was generated from the original lists with the following command:
 ##! awk '/^([a-z0-9_-]+)/ { x = $0; gsub(/[@~]/, "", x); if (length(x) > 3) print }' 93210{0,5}.data
 
-##! The heuristic for applying the `{{space-or-redirect}}` is to apply it to commands that
-##! are equal to plain English words. 
-##! The script util/fp-finder/spell.sh can help here.
+##! # check entries for English words and suffix those
+##! original="$(grep -vE '^(?:#|$)' regex-assembly/include/unix-shell-4andup-with-params.ra)"
+##! english="$(util/fp-finder/spell.sh -m -e regex-assembly/include/unix-shell-4andup-with-params.ra)"
+##! result=""
+##! while read -r oword; do
+##!   found=0
+##!   while read -r eword; do
+##!     if [ "${oword}" = "${eword}" ]; then
+##!       result="${result}${oword}{{space-or-redirect}}\n"
+##!       found=1
+##!       break
+##!     fi
+##!   done <<<${english}
+##!   if [ ${found} -eq 0 ]; then
+##!     result="${result}${oword}\n"
+##!   fi
+##! done <<<${original}
+
+##! sed -i '' -e '36,$d' regex-assembly/include/unix-shell-4andup-with-params.ra
+##! echo ${result} | sort | uniq >> regex-assembly/include/unix-shell-4andup-with-params.ra
+
+##! The two `shell` entries match `$SHELL` and `${SHELL}`.
 
 ##! These patterns are approximations of the patterns used by the cmdline
 ##! processor for `@` and `~`.
-##!> define space-or-redirect [\s<>]
+##!> define space-or-redirect [\s<>&|]
 ##!> define no-space [^\s]
 
 adduser
```

Comment:
The perl binaries HEAD and POST should be listed here (to make it consistent with GET being listed in the corresponding "upto3" file.

Reply:
Done.